src/__tests__/City.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import Adapter from 'enzyme-adapter-react-16';
```

Comment:
- [ ] Please fix this error as the previous reviewer requested:
![image](https://github.com/mohamedgamaleldin999999/metrics-app/assets/91486791/a816d1ed-8b7c-4f96-9812-f23dc5310d81)

Reply:
@tugsem thanks my friend.

the tests are passing on my local machine. Can you help me on why I am facing this problem?